submissions/lillianNwaoha/html-forms/signup.html

```diff
@@ -0,0 +1,94 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>product</title>
+    <link rel="stylesheet" href="../css/signup.css">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+<link href="https://fonts.googleapis.com/css2?family=Montserrat&display=swap" rel="stylesheet">
+</head>
+
+<body>
+        <nav>
+           <div class="logo"> 
+            <img src="/html/images/logo.PNG" alt="">
+            </div>
+        <ul class="nav-link">
+            <li><a href="../html/product.html">Product Page</a></li>
+            <li><a href="../html-forms/contact.html">Contact Us</a></li>
+            <li><a href="../html-forms/signup.html">Sign-Up</a></li>
+        </ul>
+    </nav>
+
+    <div class="top">
+        <h1>Register your Account</h1>
+        <p>Registering an account gives you access to sweet promo and events available to only registered customers.</p>
+    </div>
+
+    <div class="form">
+       <p>Already got an account? <a href="#signin">Sign in</a></p>
+    </div>
+    <div class="forms">
+       <form action="">
+       <fieldset>
+               <label for="first name">
+                  First Name</label><br> <input type="text"><br>
```

Comment:
Remove all `br` elements in this file and style it with css @Lillian-1

Reply:
done